modules/core_project_factory/versions.tf

```diff
@@ -15,12 +15,30 @@
  */
 
 terraform {
-  required_version = ">=0.12.6"
+  required_version = ">= 0.13"
 
   required_providers {
-    google      = ">= 3.50, < 4.0"
-    google-beta = ">= 3.50, < 4.0"
-    null        = "~> 2.1"
-    random      = "~> 2.2"
+    google = {
+      source  = "hashicorp/google"
+      version = ">= 3.50, < 4.0"
+    }
+    google-beta = {
+      source  = "hashicorp/google-beta"
+      version = ">= 3.50, < 4.0"
+    }
+    null = {
+      source  = "hashicorp/null"
+      version = "~> 2.1"
+    }
+    random = {
+      source  = "hashicorp/random"
+      version = "~> 2.2"
+    }
+  }
+  provider_meta "google" {
+    module_name = "blueprints/terraform/terraform-google-project-factory:core_project_factory/v10.0.0"
```

Comment:
We shouldn't use add the `provider_meta` for internal modules so let's drop this section.

Reply:
sg; in that case should `shared_vpc` be an internal module or should we add a readme for it?

Reply:
done